static/app/views/alerts/rules/issue/messagingIntegrationModal.tsx

```diff
@@ -13,11 +13,11 @@ import AddIntegrationRow from 'sentry/views/alerts/rules/issue/addIntegrationRow
 import {IntegrationContext} from 'sentry/views/settings/organizationIntegrations/integrationContext';
 
 type Props = ModalRenderProps & {
-  headerContent: React.ReactElement<any, any>;
+  headerContent: string;
```

Comment:
nit: `React.ReactNode` - which is: `string | null | JSX.Element` (basically, it's a catch all for how you're using this variable and the `bodyContent` variable. and is closer to the `React.ReactElement` type from before)